test/functional/tests/initialize/test_initialize_load.py

```diff
@@ -0,0 +1,360 @@
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_load_x_to_one_without_params(cache_mode, cleaning_policy, cache_line_size, core_number):
+    f"""
+        title: Initialize test for loading CAS with 1 cache and {core_number} cores.
+        description: |
+          Verify that loading cache configurations works properly in every mode
+          with 1 cache and {core_number} cores. Use minimal number of possible parameters
+          with load command.
+        pass_criteria:
+          - All test steps complete without errors.
+          - No data corruption is found.
+    """
+    with TestRun.step(f"Prepare 1 cache and {core_number} core devices"):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(2, Unit.GibiByte)])
+        cache_dev = cache_dev.partitions[0]
+        core_dev = TestRun.disks['core']
+        core_devices = []
+        core_size = []
+        for i in range(0, core_number):
+            core_size.append(Size(4, Unit.GibiByte))
+        core_dev.create_partitions(core_size)
+        for i in range(0, core_number):
+            core_devices.append(core_dev.partitions[i])
+
+    with TestRun.step(f"Start cache with {core_number} cores."):
+        cache = casadm.start_cache(cache_dev, cache_mode, cache_line_size, force=True)
+        core = []
+        for i in range(0, core_number):
+            core.append(cache.add_core(core_devices[i]))
+        caches_count = len(casadm_parser.get_caches())
+        if caches_count != 1:
+            TestRun.fail(f"Expected caches count: 1; Actual caches count: {caches_count}.")
+        cores_count = len(casadm_parser.get_cores(cache.cache_id))
+        if cores_count != core_number:
+            TestRun.fail(f"Expected cores count: {core_number}; Actual cores count: {cores_count}.")
+
+    with TestRun.step("Configure optional parameters."):
+        cache.set_cleaning_policy(cleaning_policy)
+        if cleaning_policy == CleaningPolicy.alru:
+            alru = FlushParametersAlru()
+            alru.activity_threshold = Time(milliseconds=1000)
+            alru.flush_max_buffers = 10
+            alru.staleness_time = Time(seconds=60)
+            alru.wake_up_time = Time(seconds=5)
+            cache.set_params_alru(alru)
+        if cleaning_policy == CleaningPolicy.acp:
+            acp = FlushParametersAcp()
+            acp.flush_max_buffers = 100
+            acp.wake_up_time = Time(seconds=5)
+            cache.set_params_acp(acp)
+
+    with TestRun.step("Run FIO on cache's exported object"):
+        for i in range(0, core_number):
+            Fio().create_command() \
+                .io_engine(IoEngine.libaio) \
+                .io_depth(64) \
+                .size(Size(4, Unit.GibiByte)) \
+                .read_write(ReadWrite.randrw) \
+                .block_size(Size(64, Unit.MebiByte)) \
```

Comment:
Is this big bs intended?

Reply:
Yes, to make it faster, but I'd change it.